ocm_python_wrapper/cluster.py

```diff
@@ -240,6 +240,8 @@ def wait_for_cluster_ready(
                 wait_timeout=time_watcher.remaining_time()
```

Comment:
why do we need `                wait_timeout=time_watcher.remaining_time()` and  `time_watcher = TimeoutWatch(timeout=wait_timeout)` in line 217?